.gitignore

```diff
@@ -109,10 +82,6 @@ __MACOSX
 /RelWithDebInfo/
 /*.build/
 
-# legacy
-/lib/liblcf/
```

Comment:
the `lib/liblcf/` is still used when doing `-DPLAYER_BUILD_LIBLCF=ON`

Reply:
Imo this should be moved to ${CMAKE_BINARY_DIR} then.